docs/getting_started/authentication.rst

```diff
@@ -52,7 +52,7 @@ of information:
             "personal use script" for the desired `developed application
             <https://www.reddit.com/prefs/apps/>`_
 
-:client_secret: The client secret is the 27-character string listed adjacent to
+:client_secret: The client secret is the 30-character string listed adjacent to
```

Comment:
Maybe we should say at least 27-character string, as mine are all 27 characters.